docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
+
+MetalK8s provides a stable, easy-to-deploy base for you to test Zenko in a
+live Kubernetes cluster environment.
+
+
+Clone or Copy the MetalK8s Git Repo
+===================================
+
+Either log in to your primary machine (any of the nodes can be the primary)
+and clone the metal-k8s repo directly from GitHub:
+
+.. code-block:: shell
+
+   $ git clone https://github.com/scality/metal-k8s
+
+or put the metal-k8s project onto the machine using sftp:
+
+.. code-block:: shell
+
+  $ sftp centos@10.0.0.1
+  Connected to 10.0.0.1
+  sftp> put -r metal-k8s
+
+Define an Inventory
+===================
+
+Each server must be configured in an inventory file that identifies servers
+and their basic configuration, including masters, etcds, and nodes.
+
+To specify the machines on which the Ansible_-based deployment system shall
+install MetalK8s, you must provide an *inventory*. This inventory contains a
+*hosts* file that lists all hosts in the cluster and *kube-node.yml*, a
+configuration file.
+
+.. _Ansible: https://www.ansible.com
+
+To create an inventory:
+
+1. Log in to the master machine and navigate to the metal-k8s repo.
+
+2. Create a directory inside the metal-k8s directory (for example,
+   :file:`inventory/quickstart-cluster`) in which the inventory will
+   be stored. Change to that directory.
+
+   .. code-block:: shell
+
+     $ cd metal-k8s
+     $ mkdir -p inventory/quickstart-cluster
+     $ cd inventory/quickstart-cluster/
+
+3. Create the :file:`hosts` file, which lists all hosts.
+
+   .. code-block:: ini
+
+     node-01 ansible_host=10.0.0.1 ansible_user=centos
+     node-02 ansible_host=10.0.0.2 ansible_user=centos
+     node-03 ansible_host=10.0.0.3 ansible_user=centos
+     node-04 ansible_host=10.0.0.4 ansible_user=centos
+     node-05 ansible_host=10.0.0.5 ansible_user=centos
+
+     [kube-master]
+     node-01
+     node-02
+     node-03
+     node-04
+     node-05
+
+     [etcd]
+     node-01
+     node-02
+     node-03
+     node-04
+     node-05
+   
+     [kube-node]
+     node-01
+     node-02
+     node-03
+     node-04
+     node-05
+   
+     [k8s-cluster:children]
+     kube-node
+     kube-master
+
+   Change the host names, IP addresses, and user names to conform to your infrastructure.
+   For example, if your servers are named "server1", "server2", and "server3", copy the code block
+   above and replace ALL instances of "node-0" with "server".
+
+4. Create a :file:`group_vars` subdirectory in the directory you created in
+   step 2 (the same directory as the :file:`hosts` file).
+
+   .. code-block:: shell
+
+    $ mkdir group_vars ; cd group_vars
+
+5. Create a file, :file:`kube-node.yml`, in the :file:`group_vars`
+   subdirectory of the inventory. This file declares how to set up storage
+   (in the default configuration) on hosts in the *kube-node* group; that is,
+   hosts on which pods shall be scheduled:
+
+   .. code-block:: yaml
+
+     metalk8s_lvm_default_vg: False

```

Comment:
Why?

Reply:
Salim suggested it. My build was not building. This seemed to move me a little closer to built.

Reply:
removed.